.github/scripts/score_and_notify.py

```
@@ -717,6 +884,22 @@ def main():
+                validation = validate_nuclear_event(title, summary)
+                if not validation.get("confirmed", False):
+                    demote_to = validation.get("demote_to") or "major"
+                    category = "major" if demote_to == "major" else "digest"
+                    if category == "major":
+                        score = max(MAJOR_THRESHOLD, min(score, NUCLEAR_THRESHOLD - 1))
+                    else:
+                        score = max(DIGEST_THRESHOLD, min(score, MAJOR_THRESHOLD - 1))
+                    print(f"  [AI DEMOTE] Nuclear candidate demoted to {category}: {validation.get('reason', 'validation failed')}")
+                else:
+                    print(f"  [AI PASS] Nuclear validation passed: {validation.get('reason', 'confirmed state-change')}" )
+            except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)